tests/test_csc.py

```diff
@@ -187,23 +187,28 @@ async def testSetPowerTooHigh(self):
                 b".0152rFanSpd3d7\r": b"#01520rFanSpd30000BC\r",
                 b".0153rFanSpd4d9\r": b"#01530rFanSpd40000BE\r",
             }
-            await harness.remote.cmd_enable.set_start(timeout=STD_TIMEOUT)
-            await harness.remote.cmd_startCooling.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_enable.set_start(timeout=STD_TIMEOUT)
+            self.csc.kiloarcModel.warmupPeriod = 15
+            await self.remote.cmd_startCooling.set_start(timeout=STD_TIMEOUT)
             await asyncio.sleep(3)  # Sleep while we wait for chiller to start chilling.
-            await harness.remote.cmd_powerLightOn.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_powerLightOn.set_start(timeout=STD_TIMEOUT)
             await asyncio.sleep(5)
-            self.assertEqual(harness.csc.kiloarcModel.component.bulbState, 800)
+            self.assertEqual(self.csc.kiloarcModel.component.bulbState, 800)
             with salobj.assertRaisesAckError():  # (result_contains="too high"):
-                await harness.remote.cmd_setLightPower.set_start(
+                await self.remote.cmd_setLightPower.set_start(
```

Comment:
Why is `result_contains` commented out? If it is valid then please enable it. Otherwise please delete it.